src/var/timeoutSet.js

```diff
@@ -0,0 +1,4 @@
+define(function() {
+	/* global setTimeout: false */
+	return window.setTimeout || setTimeout;
```

Comment:
It looks weird without context, i would suggest add descriptive comments to each of those new files